### PR TITLE
Fix style and layout typo and improve style, fix check permissions for page header.

### PR DESCRIPTION
**Pull request by KamilSupera**

### CODE REVIEW

File: src/components/Layout/PageHeader.vue [hunk 1]
```diff
@@ -60,7 +60,7 @@ export default {
         }),
 
         isLoggedIn() {
-            return this.user !== undefined;
+            return this.user;
```
Comment (IwoHerka) [inline src/components/Layout/PageHeader.vue]: Why did you change current implementation?

Comment (KamilSupera) [reply src/components/Layout/PageHeader.vue]: Because it blocks login/register page after logout.

Comment (IwoHerka) [reply src/components/Layout/PageHeader.vue]: Ok, why?

**Review by IwoHerka — APPROVED** [review]
